From: Christian Kastner <ckk@debian.org>
Date: Wed, 29 Jul 2020 17:26:33 +0200
Subject: matplotlib 3.3 fix

The 'warn' and 'force' arguments to matplotlib.use() have been
deprecated in 3.1, and removed in 3.3.

Origin: https://github.com/scikit-learn/scikit-learn/pull/17774
---
 sklearn/conftest.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sklearn/conftest.py b/sklearn/conftest.py
index d38e45f..5dadb7e 100644
--- a/sklearn/conftest.py
+++ b/sklearn/conftest.py
@@ -15,7 +15,7 @@ def pyplot():
         The ``matplotlib.pyplot`` module.
     """
     matplotlib = pytest.importorskip('matplotlib')
-    matplotlib.use('agg', warn=False, force=True)
+    matplotlib.use('agg')
     pyplot = pytest.importorskip('matplotlib.pyplot')
     yield pyplot
     pyplot.close('all')
